massa-execution-exports/src/settings.rs

```diff
@@ -102,6 +102,8 @@ pub struct ExecutionConfig {
     pub max_execution_traces_slot_limit: usize,
     /// Where to dump blocks
     pub block_dump_folder_path: PathBuf,
+    /// Max recursive calls depth in SC
+    pub max_recursive_calls_depth: usize,
```

Comment:
Could we use a u32 or u16 here so we known the size in advance?

Reply:
done